### PR TITLE
Fix lint rules in version 0.50

### PR DESCRIPTION
**Pull request by lahin31**

### CODE REVIEW

File: website/versioned_docs/version-0.50/animated.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Animated
 original_id: animated
 ---
 
-The `Animated` library is designed to make animations fluid, powerful, and easy to build and maintain. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and simple `start`/`stop` methods to control time-based animation execution.
+The `Animated` library is designed to make animations fluid, powerful, build and maintain. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and `start`/`stop` methods to control time-based animation execution.
```
Comment (rickhanlonii) [inline website/versioned_docs/version-0.50/animated.md]:
```suggestion
The `Animated` library is designed to make animations fluid, powerful, and painless to build and maintain build and maintain. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and `start`/`stop` methods to control time-based animation execution.
```

File: website/versioned_docs/version-0.50/animated.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ There are two value types you can use with `Animated`:
 `Animated` provides three types of animation types. Each animation type provides a particular animation curve that controls how your values animate from their initial value to the final value:
 
 - [`Animated.decay()`](animated.md#decay) starts with an initial velocity and gradually slows to a stop.
-- [`Animated.spring()`](animated.md#spring) provides a simple spring physics model.
+- [`Animated.spring()`](animated.md#spring) provides a spring physics model.
```
Comment (rickhanlonii) [inline website/versioned_docs/version-0.50/animated.md]:
```suggestion
- [`Animated.spring()`](animated.md#spring) provides a basic spring physics model.
```

File: website/versioned_docs/version-0.50/animated.md (hunk 1 above)
Comment (rickhanlonii) [inline website/versioned_docs/version-0.50/animated.md]:
```suggestion
The `Animated` library is designed to make animations fluid, powerful, and painless to build and maintain. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and `start`/`stop` methods to control time-based animation execution.
```

**Comment by rickhanlonii** [comment]
Ooops I think I reviewed this twice 🙈

**Comment by lahin31** [comment]
I changed this two things.

**Comment by lahin31** [comment]
Sorry for all of this. Please check.

**Comment by rickhanlonii** [comment]
No need to apologize, thanks for a great change!

**Comment by lahin31** [comment]
Welcome.

**Pull request merged**